src/Filament/Pages/Settings/ManageCachet.php

```diff
@@ -98,6 +100,18 @@ public function form(Form $form): Form
                                 ->suffix(__('cachet::settings.manage_cachet.recent_incidents_days_suffix_days'))
                                 ->hidden(fn (Get $get) => $get('recent_incidents_only') !== true),
                         ]),
+
+
+                    Forms\Components\Grid::make(2)
```

Comment:
When enabling the API, we should make sure that `api_protected` is set to `false` again by default.